loki/transformations/single_column/vector.py

```diff
@@ -140,12 +140,8 @@ def extract_vector_sections(cls, section, horizontal):
                 # we need to prevent that the whole 'else_body' is wrapped in a section,
                 # as 'Conditional's rely on the fact that the first element of the 'else_body'
                 # (if 'has_elseif') is a Conditional itself
```

Comment:
Maybe rephrase this comment/explanation?

```suggestion
                # we need to prevent that all (possibly nested) 'else_bodies' are completely wrapped as a section,
                # as 'Conditional's rely on the fact that the first element of each 'else_body'
                # (if 'has_elseif') is a Conditional itself
```